summarization_utils/toy_datasets.py

```diff
@@ -980,7 +977,94 @@ def format_prompts(self, prompt_tuples: List[Tuple[str, ...]]):
             for name, city, job, query in prompt_tuples
         ]
 
+class ToyCodeLoopTemplate(TemplaticDataset):
+    OP_NAMES = [
+        "factorial_inorder",
+        "triangular_numbers",
+        "exponentiated_numbers",
+        "subtracted_numbers",
+        "divided_numbers",
+    ]
+
+    VARS = [
+        "a",
+        "b",
+        "c",
+        "x",
+        "y",
+        "z",
+        "num",
+        "val",
+    ]
+    
+    INITIAL_VALS = [
+        "0",
+        "1",
+        # "2",
+        # "3",
+        # "4",
+        # "5",
+        # "6",
+        # "7",
+        # "8",
+        # "9",
+    ]
+
+    ANSWER_DICT = {
+        "factorial_inorder": "*",
+        "triangular_numbers": "+",
+        "exponentiated_numbers": "**",
+        "subtracted_numbers": "-",
+        "divided_numbers": "/",
+    }
+
+    COMMENTS = [
+        ""
+        #"# this is a comment explaining the code\n",
```

Comment:
I think having commented out code in utils is bad practice. I would favour removing this entirely